packages/asset-uploader/server.js

```diff
@@ -1 +1 @@
-module.exports = require("./lib/index.server");
+export * from "./lib/index.server";
```

Comment:
This doesn't adhere to "native ESM" requirement - you need an extension here.

Reply:
I bet it's true for all modules. Though it's fixable with flag (or loader in the future). We can figure this out later as well.

Reply:
ohh, really? we are using export * all over the place

Reply:
It's about explicit extension in import source "./lib/index.server.js" etc. To fix it need probably to switch from tsc but let's leave it for another PR.

Reply:
`tsc` should be able to output explicit extensions with `moduleResolution: node16` - just be cautious about this change for reasons outlined [here](https://github.com/webstudio-is/webstudio-designer/pull/493#discussion_r1029055092). Note that this, most likely, would require you to add explicit extensions to every relative import in the codebase (annoying but worthwhile in the long run)